src/elements/wrappers/ContentExplorer.js

```diff
@@ -5,6 +5,8 @@
  */
 
 import * as React from 'react';
+// TODO switch to createRoot when upgrading to React 18
```

Comment:
do we need a ticket to track this one

Reply:
It will break on its own when we head to 19